catalog.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   
+    <title>Каталог</title>
+</head>
+<body>
+    <header> 
+          
+        <nav> 
+            <a href="index.html"><!--Logo-->
+                <img src="https://via.placeholder.com/160x65" width="160" height="65" alt="Логотип">
+            </a>
+            <ul>
+              <li><a href="#">Студия</a></li>
+              <li><a href="#">Клиенты</a></li>
+              <li><a href="catalog.html">Магазин</a></li>
+              <li><a href="#">Контакты</a></li>
+            </ul>
+            <!--icon корзина-->
+            <a href="#">Корзина</a>
+          </nav>
+          <h2>Магазин готовых шаблонов</h2>
+    </header>
+
+    <main>
+
+        <section>
+            <h2>Фильтр товаров</h2>
+                
+            <form action="https://echo.htmlacademy.ru/courses" method="post" oninput="daysoutput.value=dayscount.value"><!--Двойной ползунок-->
+                <label>Стоимость: </label>
+                    <input type="range" id="dayscount" name="dayscount" min="0" max="15000" step="1">
+                        <ul>
+                            <li>От <output name="daysoutput">0</output></li>
+                            <li>До </li>
+                       </ul>
+            </form>
+            
+            <form method="get" action="https://echo.htmlacademy.ru">
+               
+                <fieldset>
+                    <legend>Сетка:</legend>
+                    <ul>
+                        <li>
+                          <input type="radio" name="Adaptive" id="filter-Adaptive" checked>
+                            <label for="filter-Adaptive">Адаптивная</label>
+                        </li>                  
+                        <li>
+                            <input type="radio" name="Fixed" id="filter-Fixed" >
+                                <label for="filter-Fixed">Фиксированная</label>
+                        </li>
+                        <li>
+                            <input type="radio" name="Rubber" id="filter-Rubber" >
+                                <label for="filter-Rubber">Резиновая</label>
+                        </li>
+                    </ul>
+                </fieldset>
+                
+                <fieldset>
+                    <legend>Особенности:</legend>
+                        <ul>
+                            <li>
+                                <input type="checkbox" name="Slider" id="filter-Slider" checked>
+                                    <label for="filter-Slider">Слайдер</label>
+                            </li>
+                        
+                            <li>
+                                <input type="checkbox" name="Advantages-block" id="filter-Advantages-block " >
```

Comment:
```suggestion
                                <input type="checkbox" name="Advantages-block" id="filter-Advantages-block" >
```
Значение `id` не должно содержать пробелов.